itests/steps/autoscaler.py

```diff
@@ -283,6 +295,15 @@ def k8s_autoscaler(context):
         monitoring_enabled=False,
     )
 
+    if prevent_scale_down_after_capacity_loss:
```

Comment:
I appreciate that this isn't as nice as overriding the config with`staticconf.test.PatchConfiguration` but I couldn't get that to override the settings defined in `itests/environment.py`. If this is too hacky, I'll have another go at getting PatchConfiguration to work.